src/handlers/api.ts

```diff
@@ -0,0 +1,204 @@
+import awsServerlessExpress from "aws-serverless-express";
```

Comment:
I'm not too familiar with serverless express but it sounds interesting, what does it give that plain serverless does not?

Reply:
I'm not too familiar with it  either 😁

The short answer is that I don't think Express gives you anything particularly special over standard AWS Lambda/Serverless.

[`aws-serverless-express`](https://github.com/awslabs/aws-serverless-express) is just an easy way to get [Express](https://expressjs.com/) to work with AWS lambda.

It means if you're used to writing apps using Express you can continue doing so. For example by creating and using Express middleware like [`authorizer`](https://github.com/connorads/lockbot/pull/58/files/c05e95cdc873861aef2376be65fcd163b10d2700#diff-1ee8d84ace45ce9c1bc3d972b0263f4bR59).

The only reason I chose to use Express was because [Bolt](https://slack.dev/bolt-js/concepts), the Slack library I'm using in the other handler in this project, also uses Express so I thought I'd use it for consistency.

There's other newer Node web frameworks on the block like [Koa](https://github.com/koajs/koa) and [Fastify](https://github.com/fastify/fastify) but I've only really dabbled in AWS Lambda and Express thus far ...